psutil/__init__.py

```diff
@@ -744,6 +744,15 @@ def cpu_affinity(self, cpus=None):
             else:
                 self._proc.cpu_affinity_set(list(set(cpus)))
 
+    # Linux and OSX only
+    if hasattr(_psplatform.Process, "environ"):
+        def environ(self):
+            """The process environment variables as a dict.
+            Note: this usually reflects the state when the process was started.
+            Later changes done by the process might not show up here.
+            """
```

Comment:
If it is true that on all platforms environ updates are not reflected then I would just make that clear.